client/me/help/main.jsx

```diff
@@ -64,7 +64,7 @@ class Help extends React.PureComponent {
 				),
 			},
 			{
-				link: localizeUrl( 'https://en.support.wordpress.com/start/' ),
+				link: 'https://en.support.wordpress.com/start/',
```

Comment:
Not sure if intentional (probably is, matches others links in `helpfulResults`.)